frameworks/elastic/tests/test_tls.py

```diff
@@ -24,8 +24,8 @@ def service_account():
     sdk_cmd.run_cli(
         "security org groups add_user superusers {name}".format(name=name))
     yield name
-    sdk_security.delete_service_account(
-        service_account_name=name, service_account_secret=name)
+    sdk_security.delete_service_account(name)
+    sdk_security.delete_secret(name)
```

Comment:
This is an unexpected change. Would it make sense to call `delete_secret` from within `delete_service_account` instead of relying on the user to do both? Do we ever have a service account without a secret?

Reply:
There are cases where secrets aren't tied to service accounts, but usually a service account has a secret with it, so I opted to separate the functions but call `delete_secret` from within `delete_service_account`